src/main/java/core/basesyntax/impl/FruitReturn.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.Returner;
+import core.basesyntax.Storage;
+import core.basesyntax.model.Fruit;
+import java.time.LocalDate;
+
+public class FruitReturn implements Returner<Fruit> {
+    private static final LocalDate NOW_DATE = LocalDate.now();
+
+    @Override
+    public void returnFruit(Fruit fruit) {
+        if (Storage.fruitStorage.peek() != null && FruitCounter.fruitsSold >= fruit.amount) {
+            if (fruit.getExpDate().isBefore(NOW_DATE)) {
+                return;
+            }
+            Storage.fruitStorage.peek().amount += fruit.amount;
+        }
```

Comment:
Here you're making the 'customer' return fruit to the 'store' when certain conditions are met, but I think we just need to take the data about what was returned from the CSV file and process it just like the two other operations (supply & buy)

Reply:
<3 @AnaLanda